packages/augur-ui/src/modules/market/components/common/comments/three-box-comments.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { use3box } from 'utils/use-3box';
+import Comments from '3box-comments-react';
+import { SecondaryButton } from 'modules/common/buttons';
+import { useAppStatusStore } from 'modules/app/store/app-status';
+import { useAppStatus } from 'modules/app/store/app-status-hooks';
+
+interface ThreeBoxCommentsProps {
+  adminEthAddr: string;
+  provider: any;
+  marketId: string;
+}
+
+const ThreeBoxComments = ({
+  adminEthAddr,
+  provider,
+  marketId,
+}: ThreeBoxCommentsProps) => {
+  const { initialized3box } = useAppStatusStore();
+  const { setInitialized3Box } = useAppStatus();
```

Comment:
this is not how we use useAppStatus. we should never use that outside of the appStatus component. instead we want to modify the line above to say `const { initialized3box, actions: { setInitialized3Box } } = useAppStatusStore();`

Reply:
Ohhhh. Now it makes sense xD